client_test.go

```diff
@@ -8,10 +8,18 @@ import (
 )
 
 var client = NewClient("localhost:6379", "test_client", MakeStringPtr("SUPERSECRET"))
+var dummy = client.FlashAll()
 
 var defaultDuration, _ = time.ParseDuration("1h")
 var tooShortDuration, _ = time.ParseDuration("10ms")
 
+func (client *Client) FlashAll() (err error) {
+	conn := client.Pool.Get()
+	defer conn.Close()
+	_, err = conn.Do("FLUSHALL")
```

Comment:
If FLUSHALL is not supported by redigo, I would suggest opening a PR there :)

Reply:
@gkorland @RoeyPrat 
Not sure why exactly the coverage is down but we should change the limitation on this or make a one-time exception

Reply:
It should not block you from merging. 
If circle ci passes and a maintainer gives you a +1, you should be able to merge.

Reply:
> If circle ci passes and a maintainer gives you a +1, you should be able to merge.
@RoeyPrat  Would you do me the honor?

Reply:
In my opinion, the commit messages are uninformative. see comment https://github.com/RedisTimeSeries/redistimeseries-go/pull/29#issuecomment-524808098

Reply:
@RoeyPrat Done!

Reply:
> In my opinion, the commit messages are uninformative. see comment [#29 (comment)](https://github.com/RedisTimeSeries/redistimeseries-go/pull/29#issuecomment-524808098)

@RoeyPrat  We are going to squash this anyway and have one comment for it all.

Reply:
![image](https://user-images.githubusercontent.com/3721069/63698418-7c86ce80-c827-11e9-9003-ec84cbe3b705.png)
these are the commits I'm seeing right now. 
1. is RTS the acronym of redis timeseries module? 
2. "minor changes" is, in my opinion, not an informative commit message.